src/extension.ts

```diff
@@ -41,54 +41,41 @@ const formatText = (text: string) => `\r${text.split(/(\r?\n)/g).join("\r")}\r`;
 	if (code) {
-		vscode.window.showErrorMessage("ERROR: `probe-rs-debugger` exited with an unexpected code: ".concat(`${code}`).concat(actionHint));
+		vscode.window.showErrorMessage("ERROR: `probe-rs-debug` exited with an unexpected code: ".concat(`${code}`).concat(actionHint));
 	} else if (signal) {
-		vscode.window.showErrorMessage("ERROR: `probe-rs-debugger` exited with signal: ".concat(`${signal}`).concat(actionHint));
+		vscode.window.showErrorMessage("ERROR: `probe-rs-debug` exited with signal: ".concat(`${signal}`).concat(actionHint));
 	}
 }
 
 // Messages to be sent to the debug session's console. 
-// Anything sent before or after an active debug session is silently ignored by VSCode. 
-// Any local (generated directly by this extension) messages MUST start with 'ERROR', or 'INFO' , 'WARN', `DEBUG`, or `TRACE` to match the RUST log behaviour. 
-// Any local messages that start with `CONSOLE` will ALWAYS be logged.
-// Any messages that come from the `probe-rs-debugger` STDERR will always be logged, and will already conform with the RUST LOG setting.
+// Any local (generated directly by this extension) messages MUST start with 'ERROR', or 'probe-rs-debug' , or `DEBUG`. 
+// Any messages that start with `ERROR` or `probe-rs-debug` will ALWAYS be logged.
+// Any messages that come from the `probe-rs-debug` STDERR will always be logged.
 function logToConsole(consoleMesssage: string, fromDebugger: boolean = false) {
 	console.log(consoleMesssage); // During VSCode extension development, this will also log to the local debug console
 	if (fromDebugger) {
-		// Any messages that come directly from the debugger, are assumed to be RUST_LOG messages and should be logged to the console.
-		vscode.debug.activeDebugConsole.appendLine(consoleMesssage);
-		// The one exception is RUST_LOG messages of the `error` variant. These deserve to be shown as an error message in the UI also.
+		// RUST_LOG messages of the `error` variant. These deserve to be shown as an error message in the UI also.
 		// This filter might capture more than expected, but since RUST_LOG messages can take many formats, it seems that this is the safest/most inclusive.
 		if (consoleMesssage.includes("ERROR")) {
-			vscode.window.showErrorMessage("`probe-rs-debugger`: " + consoleMesssage);
+			vscode.window.showErrorMessage(consoleMesssage);
+		} else {
+			// Any other messages that come directly from the debugger, are assumed to be relevant and should be logged to the console.
+			vscode.debug.activeDebugConsole.appendLine(consoleMesssage);
 		}
-	} else if (consoleMesssage.includes('CONSOLE')) {
+	} else if (consoleMesssage.includes('probe-rs-debug')) {
 		vscode.debug.activeDebugConsole.appendLine(consoleMesssage);
 	} else {
-		switch (probeRsLogLevel) {
-			case 'Error': // ONLY log Error messages
-				if (consoleMesssage.includes('ERROR')) {
+		switch (consoleLogLevel) {
+			case 'Debug': //  Log Info, Error AND Debug
```

Comment:
i would like it very much if this was a constant enum variant.

Reply:
Done :)